index.js

```diff
@@ -16,12 +16,14 @@ function create(options) {
     const _port = redirectPort === 443 ? '' : (': ' + redirectPort)
 
     return function redirectSSL(req, res, next) {
-        const _isHttps = isHTTPS(req, xForwardedProto)
-        const shouldRedirect = _isHttps === false || (redirectUnknown && _isHttps === null) && redirect
-        if (shouldRedirect) {
-            const ـredirectURL = 'https://' + (redirectHost || req.headers.host) + _port + req.url
-            res.writeHead(statusCode, { Location: ـredirectURL })
-            return res.end()
+        if (redirect) {
```

Comment:
Thanks for this PR :) What's your idea if we make condition shorter like this: ```
if (!redirect) { 
  return
}
```

Reply:
Would it be repeating what L29 does?

Reply:
Not exactly. Never mind was just a styling concern :)